examples/core/instancing/app.js

```diff
@@ -161,6 +168,48 @@ class AppAnimationLoop extends AnimationLoop {
   }
 }
 
+class TimerElement {
```

Comment:
Would be awesome if this could eventually be replaced with a standard Stats object and widget, would be nice showcase of that API...

Reply:
I'll be working on that next. This update is doubling as a POC of what I'd need from the Stats and StatsWidget in probe.gl.